DatadogSessionReplay/Tests/Recorder/ViewTreeSnapshotProducer/ViewTreeSnapshot/NodeRecorders/UIImageResourceTests.swift

```diff
@@ -20,7 +20,7 @@ final class UIImageResourceTests: XCTestCase {
         let image = createSinglePixelImage()
         let imageResource = UIImageResource(image: image, tintColor: nil)
 
-        XCTAssertNotEqual(imageResource.calculateIdentifier(), "")
+        XCTAssertEqual(imageResource.calculateIdentifier().count, 32)
```

Comment:
/question Could you explain why we cannot assert the value? The tint color will be different based on the OS version? If so, I wonder if the tint color should actually be part of the identifier 🤔

Reply:
the image hash will be different. Different iOS versions produce slightly different data, hence different hash.

We could potentially lock it in for currently selected iOS/Simu